app/src/scripts/convert.js

```diff
@@ -82,6 +82,19 @@ function convertToWebm(opts) {
   });
 }
 
+// Should be similiar to the Gif generation
+function convertToApng(opts) {
+  return Promise.resolve().then(() => {
+    return convert(opts.outputPath, opts, [
+      '-i', opts.filePath,
```

Comment:
The filter_complex throws errors in the console, not sure why tbh. The convert function just executes `converter.catch(reject);`without any real error message